File: URI/2547.py
while(True):
	try:
		N, min, max = list(map(int,input().split()))
	except EOFError:
		break

	qtd = 0
	for i in range(N):
		num = int(input())

		if num <= max and num >= min:
			qtd += 1

	print(qtd)

File: URI/C++/2547.cpp
#include<iostream>
using namespace std;

int main(){
	int N, min, max, qtd, num;

	while(cin >> N >> min >> max){
		qtd = 0;

		for(int i=0; i<N; i++){
			cin >> num;

			if(num >= min && num <= max)
				qtd++;
		}

		cout << qtd << endl;
	}

	return 0;
}